OpenRA.Mods.Common/Widgets/CheckboxWidget.cs

```diff
@@ -54,11 +53,8 @@ public override void Draw()
 			var text = GetText();
 			var textSize = font.Measure(text);
 			var check = new Rectangle(rect.Location, new Size(Bounds.Height, Bounds.Height));
-			var state = disabled ? "checkbox-disabled" :
-						highlighted ? "checkbox-highlighted" :
-						Depressed && HasPressedState ? "checkbox-pressed" :
-						Ui.MouseOverWidget == this ? "checkbox-hover" :
-						"checkbox";
+			var baseName = IsHighlighted() ? "checkbox-highlighted" : "checkbox";
```

Comment:
This leads to missing artwork / funky results if anyone ever highlights a checkbox. Can we please definitions for `checkbox-highlighted-hover`, `checkbox-highlighted-pressed`, and `checkbox-highlighted-disabled` to the chrome.yaml?

My testcase was to hardcode `IsHighlighted` and `IsDisabled` bindings to one of the settings checkboxes.

Reply:
The checkbox is one of the most complex input widgets and currently it's not handled very well in OpenRA. I have a follow up in the works that overhauls the states and naming conventions. For now I will add the missing definitions here so the current PR doesn't leave things in a weird state.

Reply:
Actually only TD has art for all of the highlighted states (both for button and checkbox). The rest of the mods just have a highlighted state and that's all. So I kept it simple in those mods by inheriting like this:
```yaml
checkbox-highlighted-hover:
	Inherits: checkbox-highlighted

checkbox-highlighted-disabled:
	Inherits: checkbox-disabled
```